xcolor.dtx

```diff
@@ -3152,7 +3143,31 @@
 %
 % \subsection*{History}
 % \addcontentsline{toc}{subsection}{History}
-%
+% \subsubsection*{2023/11/11 v2.15}
+% \begin{itemize}
+% \item if a \LaTeX{} older than 2022-06-01 is detected a frozen version is used.
+%  All new features apply only if LaTeX 2022-06-01 or newer is detected 
```

Comment:
we could say here that you can force the  old version with `[=v2]` except I think it should be `[=v2.14]` unless we call this version 3